src/seam-connect/client.ts

```diff
@@ -26,12 +26,20 @@ export interface SeamClientOptions {
 export const getSeamClientOptionsWithDefaults = (
   apiKeyOrOptions?: string | SeamClientOptions
 ): SeamClientOptions => {
-  const seamClientDefaults: SeamClientOptions = {
-    apiKey: process?.env?.SEAM_API_KEY,
-    endpoint: process?.env?.SEAM_API_URL || "https://connect.getseam.com",
-    workspaceId: process?.env?.SEAM_WORKSPACE_ID,
+  let seamClientDefaults: SeamClientOptions = {}
+  try {
+    // try to get defaults from environment (for server-side use)
+    seamClientDefaults = {
+      apiKey: process?.env?.SEAM_API_KEY,
+      endpoint: process?.env?.SEAM_API_URL || "https://connect.getseam.com",
+      workspaceId: process?.env?.SEAM_WORKSPACE_ID,
+    }
+  } catch (error) {
```

Comment:
How do we know we are in browser? Can we check for `process.env` instead of catching all errors, like `globalThis?.process?.env == null`

Reply:
I think globalThis is the intended solution for this. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis